_site/posts/posts.json

```diff
@@ -1,4 +1,21 @@
+    "preview": {},
+    "last_modified": "2025-11-10T21:43:28+01:00",
```

Comment:
in the list with "the rules you want to apply" why not add commas after each item? :nail_care: